packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/SynapseTableCell.integration.test.tsx

```diff
@@ -1,52 +1,43 @@
 import { render, screen } from '@testing-library/react'
 import React from 'react'
-import { SynapseConstants } from '../../../src/utils'
-import SynapseTableCell, {
-  SynapseTableCellProps,
-} from '../../../src/components/SynapseTable/SynapseTableCell/SynapseTableCell'
-import { NOT_SET_DISPLAY_VALUE } from '../../../src/components/SynapseTable/SynapseTableConstants'
-import { createWrapper } from '../../../src/testutils/TestingLibraryUtils'
-import {
-  ENTITY_HEADERS,
-  ENTITY_ID_VERSION,
-} from '../../../src/utils/APIConstants'
+import { SynapseConstants } from '../../../utils'
```

Comment:
This file was impacted by reorganizing some mocks, but I realized that it was in the old test folder and was not type-checked, so I colocated it with its component and fixed type errors. No substantial changes to test logic.